src/api/app/views/webui/package/show.html.erb

```diff
@@ -38,6 +38,13 @@
               at <%= link_to(elide(dpackage.project.name, 44), :action => :show, :controller => :package, :project => dpackage.project.name, :package => dpackage.name) %>
             </li>
         <% end %>
+        <% if @package.project != @project %>
+            <li>
+              <%= sprite_tag 'information', title: '' %>
+              Sources life in linked project
```

Comment:
Thought about that, but it sounds more like a package link. actually the sources are not really "linked" but just re-used in another project. 

Maybe 

Sources used from project $project

?